pkg/util/client.go

```diff
@@ -18,6 +18,10 @@ const (
 	orgIDHeader = "X-Scope-OrgID"
 
 	QueryTraceEndpoint = "/api/traces"
+
+	AcceptHeader        = "Accept"
```

Comment:
i think these are already defined in `/pkg/http`

Reply:
Yes, but that causes an import cycle. They could be moved here, but I think they're better suited in `pkg/http.go`. What I can do is un-export them.